File: scripts/jvm/json.scala
//> using dep com.lihaoyi::upickle::4.2.1
object Json {
  def main(args: Array[String]): Unit = {
    var s = ""
    var x = 0
    while (x < 10000) {
      s = ujson.write(ujson.Obj("a" -> ujson.Arr(x, 2, 3), "b" -> "hello"))
      x += 1
    }
    println(s)
  }
}

File: scripts/jvm/json.kt
import org.json.JSONObject
import org.json.JSONArray
fun main() {
    var s = ""
    for (x in 0 until 10000) {
        s = JSONObject().put("a", JSONArray().put(x).put(2).put(3)).put("b", "hello").toString()
    }
    println(s)
}
